bee-api/bee-rest-api/src/endpoints/routes/api/plugins/debug/white_flag.rs

```diff
@@ -66,12 +66,12 @@ pub(crate) async fn white_flag<B: StorageBackend>(
     storage: ResourceHandle<B>,
     tangle: ResourceHandle<Tangle<B>>,
     bus: ResourceHandle<Bus<'static>>,
-    message_requester: MessageRequesterWorker,
-    requested_messages: ResourceHandle<RequestedMessages>,
+    block_requester: BlockRequesterWorker,
+    requested_blocks: ResourceHandle<RequestedBlocks>,
     rest_api_config: RestApiConfig,
 ) -> Result<impl Reply, Rejection> {
     let index_json = &body["index"];
-    let parents_json = &body["parentMessageIds"];
+    let parents_json = &body["parents"];
```

Comment:
Is this change correct?

Reply:
Yes, it had been simplified to this in iota.go as well

Reply:
Should be still `parentBlockIds` according to https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/stardust-api/tips/TIP-0025/core-rest-api.yaml

Reply:
Yeah I guess they didn't change it yet, Levente is/was off these days